ic-os/components/selinux/dante/dante.fc

```diff
@@ -0,0 +1,5 @@
+/etc/danted\.conf	--	gen_context(system_u:object_r:dante_conf_t,s0)
```

Comment:
Maybe we use the hardcoded `/etc/danted.conf` as the config file for dante instead of the generated one, as the generated one is constant? And include this file as usual through IC-OS build components

Reply:
Switched to hardcoded `/etc/danted.conf` and had to extend the SELinux policies by this:
```
allow dante_t dante_conf_t:file { getattr ioctl open read };
```
At the same time, I also removed the `socks` user from sudoers.